RoslynPluginGenerator/NuGet/NuGetPackageHandler.cs

```diff
@@ -17,60 +16,17 @@ namespace SonarQube.Plugins.Roslyn
 {
     public class NuGetPackageHandler : INuGetPackageHandler
     {
-        private readonly string packageSource;
```

Comment:
Previously we used a single hard-coded package source (e.g. https://www.nuget.org/api/v2/). The code now accepts  an IPackageRepository, which could be an AggregateRepository i.e. a repo that groups together a collection of other repos.